app/server/endpoint/src/main/java/io/syndesis/server/endpoint/actuator/ConnectorsEndpoint.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.server.endpoint.actuator;
+
+import io.syndesis.common.model.connection.Connector;
+import io.syndesis.server.dao.manager.DataManager;
+import org.springframework.boot.actuate.endpoint.AbstractEndpoint;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+
+/**
+ * Defines methods for refreshing connectors without restarting the server.
+ */
+@Configuration
+@ConditionalOnProperty(value = "endpoints.connectors.enabled", havingValue = "true", matchIfMissing = true)
+public class ConnectorsEndpoint extends AbstractEndpoint<List<Connector>> {
```

Comment:
Could we use the existing `PUT /api/v1/connector` endpoint?

https://github.com/syndesisio/syndesis/blob/f29e90402c330e4e67c6103530725acec1eb739b/app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/connection/ConnectorHandler.java#L214